requirements/bootstrap-lock.txt

```diff
@@ -0,0 +1,16 @@
+#
+# This file is autogenerated by pip-compile with python 3.9
+# To update, run:
+#
+#    pip-compile --allow-unsafe --generate-hashes --output-file=requirements/bootstrap-lock.txt --unsafe-package=flit-core --unsafe-package=pip --unsafe-package=setuptools --unsafe-package=wheel requirements/bootstrap.txt
+#
+flit-core==3.7.1 \
+    --hash=sha256:14955af340c43035dbfa96b5ee47407e377ee337f69e70f73064940d27d0a44f \
+    --hash=sha256:e454fdbf68c7036e1c7435ec7479383f9d9a1650ca5b304feb184eba1efcdcef
+    # via -r requirements/bootstrap.txt
+
+# The following packages are considered to be unsafe in a requirements file:
+pip==22.2.2 \
+    --hash=sha256:3fd1929db052f056d7a998439176d3333fa1b3f6c1ad881de1885c0717608a4b \
+    --hash=sha256:b61a374b5bc40a6e982426aede40c9b5a08ff20e640f5b56977f4f91fed1e39a
+    # via -r requirements/bootstrap.txt
```

Comment:
Is there a particular reason we have these top-level `bootstrap-lock.txt` and `bootstrap-win-lock.txt` lock files? I would have assumed we would removed them as the ones that would be used are the ones in the `download-deps` directory.